packages/mobile/src/send/SendAmount.tsx

```diff
@@ -58,16 +58,21 @@ import {
   RecipientKind,
 } from 'src/recipients/recipient'
 import { RootState } from 'src/redux/reducers'
+import { PaymentInfo } from 'src/send/reducers'
+import { getRecentPayments } from 'src/send/selectors'
 import { ConfirmationInput } from 'src/send/SendConfirmation'
 import DisconnectBanner from 'src/shared/DisconnectBanner'
 import { fetchDollarBalance } from 'src/stableToken/actions'
+import { timeDeltaInHours } from 'src/utils/time'
 import { withDecimalSeparator } from 'src/utils/withDecimalSeparator'
 
 const AmountInput = withDecimalSeparator(
   withTextInputLabeling<ValidatedTextInputProps<DecimalValidatorProps>>(ValidatedTextInput)
 )
 const CommentInput = withTextInputLabeling<TextInputProps>(TextInput)
 
+const DAILY_PAYMENT_LIMIT_USD: BigNumber = new BigNumber(500)
```

Comment:
Please move this to Config.ts and just make it a `Number`

Reply:
Done.